packages/cosmo/package.py

```diff
@@ -202,28 +195,8 @@ class Cosmo(MakefilePackage):
             description='Slurm constraints for nodes requested')
 
     conflicts('+claw', when='cosmo_target=cpu')
-    conflicts('+pollen', when='@5.05:5.06,org-master,master')
+    conflicts('+pollen', when='@org-master,master')
```

Comment:
c2sm-master and c2sm-features and empa-ghg need conflicts as well

Reply:
Perfect point for an other PR 👍